testplan/base.py

```diff
@@ -262,6 +262,11 @@ def parser(self):
         return self._cfg.parser(name=self._cfg.name,
                                 default_options=self._default_options)
 
+    @property
+    def runnable(self):
```

Comment:
is this used anywhere?

Reply:
I used it before, but I changed my code so it's not in use any more, I can remove it. We may also keep it, I saw in `test_plan_base.py` plan._runnable is frequently used.

Reply:
It's fine I was mostly just curious.